packages/xdl/src/Project.js

```diff
@@ -1885,7 +1885,7 @@ export async function bundleWebpackAsync(projectRoot, packagerOpts) {
   const hasWebSupport = await Web.hasWebSupportAsync(projectRoot);
   if (!hasWebSupport) {
     Web.logWebSetup();
-    return;
+    return false;
```

Comment:
Let's throw an error here as well. I think the best way to do this would be to add a new function `Web.ensureWebSupportAsync(projectRoot)`, which checks `hasWebSupportAsync` and throws an error like `new XDLError(ErrorCode.WEB_NOT_CONFIGURED, getWebSetupLogs())`.

Reply:
Will do this in the evening (CEST)!

Reply:
Done